backend/datasource/query_type_mux.go

```diff
@@ -110,7 +110,8 @@ func fallbackHandler(_ context.Context, req *backend.QueryDataRequest) (*backend
 	responses := backend.Responses{}
 	for _, q := range req.Queries {
 		responses[q.RefID] = backend.DataResponse{
-			Error: fmt.Errorf("no handler found for query type '%s'", q.QueryType),
+			Error:       fmt.Errorf("no handler found for query type '%s'", q.QueryType),
```

Comment:
Random comment. Looking at this seems like it could be either a user error -> providing a query type not supported or something wrong with the plugin. Guess it's safest to use `backend.ErrorSourcePlugin` because not sure we can distinguish between the two?

Reply:
Seems to me this is a plugin error.  The plugin should define the query types that can be provided.  The user shouldn't be able to supply any arbitrary query type.

Reply:
You're right in theory. But in practice a user can via the Grafana API /api/ds/query specify/define the queries freely and set queryType to anything :) No blocker really, lets leave it like this for now.